controllers/ssp_controller.go

```diff
@@ -197,7 +197,7 @@ func (r *sspReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res ct
 	if err != nil {
 		return ctrl.Result{}, err
 	}
-	sspRequest.Logger.V(1).Info("CR status updated")
+	sspRequest.Logger.Info("CR status updated")
```

Comment:
Can you keep these 2 log lines in debug, and add a third log line with info level at the end of this method, that will say something like: Reconciliation finished.

Reply:
actually this is done on purpose to give more information (if the log line it is there we have more info on what failed)

Reply:
ok